docs/tutorials/visualizing-node-metrics.md

```diff
@@ -109,14 +109,33 @@ substrate_ready_transactions_number 0
 
 ## Next Steps
```

Comment:
The next steps heading should be retained at the bottom of the tutorial. `## Next Steps` is how we start to say "that's all this tutorial teaches, but here are some other readings you may enjoy"

Reply:
Got it. Thanks!